framework/py/flwr/common/inflatable_test.py

```diff
@@ -40,7 +42,12 @@ def deflate(self) -> bytes:  # noqa: D102
         return add_header_to_object_body(object_body=obj_body, cls=self)
 
     @classmethod
-    def inflate(cls, object_content: bytes) -> "CustomDataClass":  # noqa: D102
+    def inflate(  # noqa: D102
+        cls, object_content: bytes, children: dict[str, InflatableObject]
+    ) -> "CustomDataClass":
```

Comment:
May I know what's `# noqa: D102` for? Plus, I am inclined to avoid duoble quotes around class name for consistency purpose. We can add `from __future__ import annotations`.

```suggestion
    def inflate(  # noqa: D102
        cls, object_content: bytes, children: dict[str, InflatableObject]
    ) -> CustomDataClass:
```